src/core/core.animator.js

```diff
@@ -2,6 +2,10 @@
 
 import helpers from '../helpers';
 
+/**
+ * @typedef { import("./core.controller") } Chart
```

Comment:
Learned something new again :)

Reply:
This is a TypeScript-specific doc. But I think we could have TypeScript generate the docs instead of the jsdoc tool and hopefully it would add the appropriate linking between classes